libcob/common.h

```diff
@@ -1248,6 +1248,51 @@ typedef struct __cob_screen {
 	int			attr;		/* COB_SCREEN_TYPE_ATTRIBUTE */
 } cob_screen;
 
+/* String structure */
+struct cob_inspect_state {
+	cob_field	    	*var;
+	unsigned char		*data;
+	unsigned char		*start;
+	unsigned char		*end;
+	unsigned char		*mark;	/* buffer to marker only: 0/1 */
+	size_t		    	mark_size;	/* size of internal marker elements, increased up to
+                                 the maximum needed (biggest target field size) */
+	size_t			    mark_min;	/* min. position of the marker set by the last initialize */
+	size_t			    mark_max;	/* max. position of the marker set by the last initialize */
+	unsigned char		*repdata;	/* contains data for REPLACING which is applied at end */
+	size_t			    repdata_size;	/* size of internal repdata buffer, increased up to
+                                   the maximum needed (biggest target field size) */
+	size_t			    size;
+	cob_u32_t		    replacing;	/* marker about current operation being INSPECT REPLACING */
+	int			        sign;
+	cob_field		    var_copy;
+};
+
+struct cob_string_state {
+	cob_field		*dst;
+	cob_field		*ptr;
+	cob_field		*dlm;
+	cob_field		dst_copy;
+	cob_field		ptr_copy;
+	cob_field		dlm_copy;
+	int		    	offset;
```

Comment:
I think so? Do you think otherwise?

In the previous code, we had `string_offset` and `unstring_offset` which were both static variables used in global functions.

Reply:
My question was not clear, let me rephrase it: Do we need a value of -1 or similar?`If not then it is likely reasonable to use size_t for an offset variable, no?

Reply:
ok, if the offset is initialized by `WITH POINTER`, then just add that information and leave as is; if count and ndlms are only used positive, then please adjust them to `unsigned int`.